### PR TITLE
Improve/drop self

### PR DESCRIPTION
**Pull request by zenangst**

### CODE REVIEW

File: Source/TutorialController.swift [hunk 1]
```diff
@@ -30,14 +30,14 @@ let MinimumMarginLateralSpace: CGFloat = 20.0
...
-    self.add(pages)
+    add(pages)
   }
 
   override public func add(viewControllers: [UIViewController]) {
     for viewController: UIViewController in viewControllers {
-      let titleLabel = self.titleLabel
-      titleLabel.text = viewController.title
-      viewController.view.addSubview(titleLabel)
+      let viewTitleLabel = titleLabel
```
Comment (3lvis) [inline Source/TutorialController.swift]: Use the same name here

Comment (zenangst) [inline Source/TutorialController.swift]: Nope then the compiler screams

Comment (3lvis) [inline Source/TutorialController.swift]: ewww

https://github.com/hyperoslo/iOS-playbook/blob/master/style-guidelines/Swift.md#constants

Comment (3lvis) [inline Source/TutorialController.swift]: Use self then

**Comment by zenangst** [comment]
@3lvis should be fixed now

**Comment by 3lvis** [comment]
Looks good! :+1:

**Pull request merged**